pilot/pkg/proxy/envoy/v2/discovery.go

```diff
@@ -155,7 +155,13 @@ func NewDiscoveryServer(
 	}
 
 	// Flush cached discovery responses whenever services configuration change.
-	serviceHandler := func(*model.Service, model.Event) { out.clearCache(&model.PushRequest{Full: true}) }
+	serviceHandler := func(*model.Service, model.Event) {
+		pushReq := &model.PushRequest{
+			Full:               true,
+			ConfigTypesUpdated: map[string]struct{}{schemas.ServiceEntry.Type: {}},
```

Comment:
I don't think this is ServiceEntry only is it?

Reply:
we use SE to represent service here

Reply:
what about Consul?